apps/lib/s_cb.c

```diff
@@ -1415,8 +1415,11 @@ int ssl_load_stores(SSL_CTX *ctx,
         vfy = X509_STORE_new();
         if (vfy == NULL)
             goto err;
-        if (vfyCAfile != NULL && !X509_STORE_load_file(vfy, vfyCAfile))
+        if (vfyCAfile != NULL && !X509_STORE_load_file(vfy, vfyCAfile)) {
```

Comment:
Not sure the changes in this file should be in a comment titled "s_client.c,*.pod: fix documentation..."